runtime/queries/cylc/highlights.scm

```diff
@@ -0,0 +1,101 @@
+(ERROR) @markup.bold
+
+[
+  (jinja2_expression)
+  (jinja2_statement)
+  (jinja2_comment)
+  (jinja2_shebang)
+] @special
+
+(include_statement
+  directive: _ @keyword.directive
+  path: _ @string.path)
+
+(comment) @comment.line
+
+(graph_section
+  name: _? @label)
+
+(task_section
+  name: (_
+    (task_name) @namespace))
+
+(top_section
+  brackets_open: _ @ponctuation.bracket
```

Comment:
there's a typo in these "ponctuation" => "punctation"

Reply:
I was actually gonna change it to "p***u***nct***u***ation" but I can make it "punctation" if you insist 👽

btw, I'm adding support to Neovim, and they have a [`check-queries.lua`](https://github.com/nvim-treesitter/nvim-treesitter/blob/master/scripts/check-queries.lua) script to catch wrong capture names. Since Helix doesn't use Lua I guess it would have to be rewritten from scratch. 😕